drizzlepac/hapsequencer.py

```diff
@@ -1377,32 +1379,33 @@ def delete_ramp_exposures(obj_list, type_of_list):
 
     Returns
     -------
-    ramp_filenames_list : list of str
-        List of ramp exposure SVM FLT/FLC filenames and their corresponding trailer filenames
+    excluded_filenames_list : list of str
+        List of ramp/quadrant exposure SVM FLT/FLC filenames and their corresponding trailer filenames
 
     """
     reported = False
     temp = []
-    ramp_filenames_list = []
+    excluded_filenames_list = []
     while obj_list:
         obj = obj_list.pop()
-        if obj.filters.lower().find('fr') == -1:
+        # If the filter name does not contain 'fr' or 'fq', then it is okay to keep.
+        if (obj.filters.lower().find('fr') == -1) and (obj.filters.lower().find('fq') == -1):
```

Comment:
I realize the previous code used `find` but I wonder if `startswith` is a better choice given that we expect the filter name to start with these characters. Then I don't know how HST filters are named and if this is relevant.